rust/exchange_rate/canister_ids.json

```diff
@@ -0,0 +1,5 @@
+{
+  "exchange_rate": {
+    "ic": "464yd-nyaaa-aaaam-qag5a-cai"
```

Comment:
Should the mainnet deployment id be included in the example? Also, what's the difference between this id and the one mentionned in README (mfwph-oqaaa-aaaam-qafsq-cai)?

Reply:
You are right, this one is not needed. The canister_id referred in README.md is for mainnet, while this one, I believe is on some testnet.